geode-assembly/src/test/java/org/apache/geode/session/tests/TomcatInstall.java

```diff
@@ -131,15 +142,20 @@ public TomcatInstall(TomcatVersion version, ConnectionType connType) throws Exce
    * within the installation's 'conf' folder, and {@link #updateProperties()} to set the jar
    * skipping properties needed to speedup container startup.
    */
-  public TomcatInstall(TomcatVersion version, ConnectionType connType, String installDir)
-      throws Exception {
+  public TomcatInstall(TomcatVersion version, ConnectionType connType, String installDir,
+      String modulesJarLocation, String extraJarsPath) throws Exception {
     // Does download and install from URL
-    super(installDir, version.getDownloadURL(), connType, "tomcat");
+    super(installDir, version.getDownloadURL(), connType, "tomcat",
```

Comment:
Refactor both "tomcat" and DEFAULT_MODULE_LOCATION pass down/up into other constructors